sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/api/Case.java

```diff
@@ -54,4 +54,5 @@
   Region region;
   HealthFacility healthFacility;
   MaternalHistory maternalHistory;
+  String facilityType;
```

Comment:
Does this change impact other tests that are using case created through API?

Reply:
Running the API feature returns pass:
![image](https://user-images.githubusercontent.com/46056983/121329519-f5ccb400-c91d-11eb-823a-2c473df354e6.png)
But we can be sure only when having regression.

Reply:
Same as above.